microsoft-edge/devtools-guide-chromium/device-mode/orientation.md

```diff
@@ -50,7 +50,7 @@ You can also set a custom orientation by dragging the **Orientation Model**.  Fo
 <!-- ====================================================================== -->
 > [!NOTE]
 > Portions of this page are modifications based on work created and [shared by Google](https://developers.google.com/terms/site-policies) and used according to terms described in the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0).
-> The original page is found [here](https://developer.chrome.com/docs/devtools/device-mode/orientation/) and is authored by Kayce Basques.
+> The original page is found [here](https://developer.chrome.com/docs/devtools/sensors#orientation) and is authored by Kayce Basques.
```

Comment:
no longer a page; now a section of page.  analyze headings outlines.  "original page" inaccurate.

Reply:
we'll open work item to redesign docs per upstream